libnavui-androidauto/src/main/java/com/mapbox/androidauto/CarAppState.kt

```diff
@@ -9,6 +12,9 @@ package com.mapbox.androidauto
  */
 sealed class CarAppState
 object FreeDriveState : CarAppState()
-object RoutePreviewState : CarAppState()
+data class RoutePreviewState(
+    val placeRecord: PlaceRecord,
+    val routes: List<NavigationRoute>,
+) : CarAppState()
```

Comment:
@kmadsen with this changes it is almost impossible to sync drop-in state with Android Auto. For example drop-in goes into route preview state first and only then tries to reverse-geocode the location (which may not succeed).

Reply:
Yes, we need a release that can be integrated: #6043.